ami/install.bash

```diff
@@ -102,5 +102,8 @@ git clone https://github.com/mesosphere/marathon.git /home/admin/marathon
 cd /home/admin/marathon
 sbt update
 
+echo -e "\n=== Install Utils ==="
+apt-get install strace
```

Comment:
Having this means we don't need the "installUtils()" as soon as we build a new AMI?

Reply:
Yep. I can remove `installUtils`

Reply:
Though, I don't think we'll ever update the 1.8 AMI so 🤷‍♂